### ISSUE TITLE
Nodes show no inbound traffic

### ISSUE THREAD

**Issue by autholykos** [dusk-network/rusk#1489]
#### Some nodes do not get blocks propagated to them

We have been noticing some nodes both in our own cluster as well as ITN's that show no inbound traffic. We shall investigate the reason. 

#### Relevant Context (optional)

One of the nodes affected in our cluster is `node-27`

**Comment by goshawk-3** [comment]
```
--------------------------------------------------------------------------------------------
Total send rate:                                     1.39Kb       938b       938b
Total receive rate:                                  2.70Kb     1.96Kb     1.96Kb
Total send and receive rate:                         4.10Kb     2.88Kb     2.88Kb
```

**Comment by goshawk-3** [comment]
```
[1097322.173408] Out of memory: Killed process 174091 (rusk) total-vm:2301620kB, anon-rss:1422836kB, file-rss:0kB, shmem-rss:0kB, UID:1000 pgtables:3036kB oom_score_adj:0
```

**Comment by goshawk-3** [comment]
Closed due to #1495
